package.json

```diff
@@ -18,8 +18,10 @@
     "build-tsc": "tsc -p tsconfig.json",
     "fix": "npm run admin -- lint --fix",
     "lint": "npm run admin -- lint",
+    "templates": "node ./bin/devkit-admin templates",
     "test": "node ./bin/devkit-admin test",
     "test:watch": "nodemon --watch packages -e ts ./bin/devkit-admin test",
+    "validate": "./bin/devkit-admin validate",
```

Comment:
This should have `node` at the start too.

Reply:
Done. I'll submit.